### PR TITLE
Early phase dev experience contributions

### PR DESCRIPTION
**Pull request by cnpryer**
Not sure if this is still relevant to contribute/if it is compatible with the updated master. However I'm submitting this PR to promote open source activity. Feel free to close this without a merge.

Will try to contribute more in the future. :(

The only real value here would be having a CONTRIBUTING.md.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# -*- mode: ruby -*-
```
Comment (Michael-Steshenko) [inline Vagrantfile:1]: can you please explain the use of this file? If not needed please edit the PR and remove it and its refrence from .gitignore

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#   CONTRIBUTING
```
Comment (Michael-Steshenko) [inline CONTRIBUTING.md:1]: awesome

File: scripts/linux-bootstrap.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+sudo apt-get update
```
Comment (Michael-Steshenko) [inline scripts/linux-bootstrap.sh:1]: great

**Review by Michael-Steshenko — COMMENTED** [review]
good stuff, please see additional comments in review

**Comment by cnpryer** [comment]
This is super old. Predates a lot of design decisions that were motivating where I was heading with this.

I'll close it out. Thanks for the response.

**Pull request closed without merging**